getting-started/templates/systemlink-values.yaml

```diff
@@ -1008,6 +1008,35 @@ argoworkflows:
 ##
 nbexecservice:
   maxNumberOfWorkflowsToSchedule: *workflowParallelism
+  ## Configure S3/MinIO access.
+  ##
+  s3:
+    ## Secret name for S3 credentials.
+    ##
+    secretName: "executions-s3-credentials"
```

Comment:
I suggest using the default: `nbexecservice-s3-credentials`

Reply:
Updated